raster-foundry/raster-foundry
# Add back project color clipping

## Overview


### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo

![image](https://user-images.githubusercontent.com/4392704/43332793-3485c55e-9197-11e8-99b5-9d77fe59b9fc.png)

### Known issues
- [ ] Debouncing is sometimes making changes not go through to the api. It happens rarely enough that I don't think it's worth addressing immediately in this pr.
- [ ] Api does not return valid histograms for COGs, so clipping does not work https://github.com/raster-foundry/raster-foundry/issues/3774
- [x] remove debugging comments

## Testing Instructions

* Go to a project with scenes in it
* click `color mode`, `corrections`, select a scene, and then `color correct`
* Adjust clipping for each color band and verify that they work as expected

Closes https://github.com/azavea/raster-foundry-platform/issues/404
